### PR TITLE
drii-cavalcanti/c03-docker01

### PR DESCRIPTION
**Pull request by drii-cavalcanti**
# Description

Exercise (e.g. C01-E01): c03-docker01

Student Name: Adriana Cavalcanti

> A brief explanation of what has been done.
creating a docker image
## Type of change

  - [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
  - [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker01/drii-cavalcanti/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.7
+RUN apk add terraform
```
Comment (denstorti) [inline classes/03class/exercises/c03-docker01/drii-cavalcanti/Dockerfile]: The solution is functional, Adriana. That's good, but your image has more than 80MB, so something else is in the final image.

#### Can you make the final docker image size be less than 80MB?

Terraform itself is around 50MB, Alpine is around 5MB.

**Tip**: Terraform comes in a zipped file, that just needs to be unzipped and moved to a folder that is in the PATH environment variable. This should result in less than 60MB.

You can find terraform here: https://www.terraform.io/downloads.html

Comment (drii-cavalcanti) [reply classes/03class/exercises/c03-docker01/drii-cavalcanti/Dockerfile]: am I supposed to use the wget command?

Comment (kikobr82) [reply classes/03class/exercises/c03-docker01/drii-cavalcanti/Dockerfile]: Not necessarily.

One option is go with a multistage build:
https://docs.docker.com/develop/develop-images/multistage-build/

Or you can use wget to download the zip file and then remove all non necessary files from the build(like wget, zip tool and the terraform.zip file)...
Hit me up if you need further explanation.

Comment (kikobr82) [reply classes/03class/exercises/c03-docker01/drii-cavalcanti/Dockerfile]: also, make sure you have the right version of terraform...:)

**Pull request closed without merging**